tests/layer_tests/pytorch_tests/test_rad2deg.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+import numpy as np
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestRad2Deg(PytorchLayerTest):
+    def _prepare_input(self, input_shape=(2, 2)):
+        return (np.random.uniform(-np.pi, np.pi, size=input_shape).astype(np.float32),)
+
+    def create_model(self):
+        class aten_rad2deg(torch.nn.Module):
+            def forward(self, x):
+                return x * (180.0 / np.pi)
+        ref_net = None  
+        return aten_rad2deg(), ref_net, "aten::mul"
```

Comment:
This is not a test for rad2deg
```suggestion
        return aten_rad2deg(), ref_net, "aten::rad2deg"
```